financial-connections/src/main/java/com/stripe/android/financialconnections/domain/PollAttachPaymentAccount.kt

```diff
@@ -18,6 +19,7 @@ import kotlin.time.Duration.Companion.seconds
 
 internal class PollAttachPaymentAccount @Inject constructor(
     private val repository: FinancialConnectionsAccountsRepository,
+    private val consumerSessionRepository: ConsumerSessionRepository,
```

Comment:
I’d recommend to rebase onto https://github.com/stripe/stripe-android/pull/8956, so that you can inject `ConsumerSessionProvider` instead.

Reply:
I'll wait for your merge then update this!